content/en/news/2019/announcing-1.4/change-notes/index.md

```diff
@@ -16,6 +16,12 @@ weight: 10
 
 ## Telemetry
 
+- **Added** experimental [in-proxy](/docs/ops/telemetry/in-proxy-service-telemetry/) telemetry reporting to Stackdriver.
+- **Added** experimental support for generating TCP service metrics [in-proxy](/docs/ops/telemetry/in-proxy-service-telemetry/).
+- **Improved** support for [in-proxy](/docs/ops/telemetry/in-proxy-service-telemetry/) Prometheus generation of HTTP service metrics (from experimental to alpha).
+- **Improved** stability for proxy metadata exchange (from experimental to alpha).
+- **Improved** egress gateway telemetry.
```

Comment:
@douglas-reid any more info on this?

Reply:
Yes.  We can link to the issues closed. I'll find them and add them as a comment here.

Reply:
Any updates?

Reply:
See suggestion below